### PR TITLE
[JENKINS-64376] Add hidden columns

### PR DESCRIPTION
**Pull request by KasperHeyndrickx**
https://issues.jenkins.io/browse/JENKINS-64376

### CODE REVIEW

**Comment by KasperHeyndrickx** [comment]
Looks like it failed to connect with windows on AWS? Is it possible to rerun this without pushing additional empty commits?

File: src/main/resources/data-tables/table.jelly [hunk 1]
```diff
@@ -35,7 +35,8 @@
     </j:if>
     <table class="table table-hover table-striped data-table ${class}"
            data-columns-definition="${model.columnsDefinition}" id="${model.id}"
-           data-table-configuration="${model.tableConfigurationDefinition}">
+           data-table-configuration="${model.tableConfigurationDefinition}"
+           style="width: 100%;">
```
Comment (uhafner) [inline src/main/resources/data-tables/table.jelly:39]: Wouldn't it make more sense to move that part to the `css` file?

**Review by uhafner — APPROVED** [review]

File: src/main/resources/data-tables/table.jelly (hunk 1 above)
Comment (KasperHeyndrickx) [reply src/main/resources/data-tables/table.jelly:39]: I tried adding it to jenkins-style.css, line 7: 
```
table.dataTable {
    border-collapse: collapse !important;
    width: 100%;
}
```
But that doesn't seem to work. The problem is that if I don't add this line in the jelly file, it gets 'width: 0px' when rendered:
```
<table data-table-configuration="{}" ... style="width: 0px;" isloaded="true">
```
I'm not sure where this is coming from. This [link](https://datatables.net/forums/discussion/51750/style-width-0px-appears-for-bootstrap4-complex-headers) seems related, but there's no real solution there. They do put width: 100% in the html file though...

Comment (uhafner) [reply src/main/resources/data-tables/table.jelly:39]: Ok, I'm not a CSS expert so let us keep it this way.

**Review by uhafner — APPROVED** [review]

**Pull request merged**